examples/hono-client-examples/src/main/java/org/eclipse/hono/devices/AmqpExampleDevice.java

```diff
@@ -135,14 +135,12 @@ private void sendTelemetryMessageWithQos2() {
                     if (ar.succeeded()) {
                         System.out.println("Telemetry message with QoS 'AT_LEAST_ONCE' sent: " + payload);
                     } else {
-                        final Throwable cause = ar.cause();
                         String hint = "";
-                        if (cause instanceof ServerErrorException
-                                && ((ServerErrorException) cause).getErrorCode() == 503) {
+                        if (ServiceInvocationException.extractStatusCode(ar.cause()) == 503) {
```

Comment:
can you also fix the naming of the methods?
*sendTelemetryMessageWithQos1* needs to be *sendTelemetryMessageWithQos0* and
*sendTelemetryMessageWithQos2* needs to be *sendTelemetryMessageWithQos1* ...

Reply:
Nice catch. I have renamed them now.